.github/workflows/tools/update-issues/main.js

```diff
@@ -0,0 +1,223 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview A workflow tool to maintain GitHub issues.
+ */
+
+const core = require('@actions/core');
+const { Issue, Milestone } = require('./issues.js');
+
+const TYPE_BUG = 'type: bug';
+const TYPE_CI = 'type: CI';
+const TYPE_CODE_HEALTH = 'type: code health';
+const TYPE_DOCS = 'type: docs';
+const TYPE_ENHANCEMENT = 'type: enhancement';
+const TYPE_PERFORMANCE = 'type: performance';
+const TYPE_QUESTION = 'type: question';
+
+const STATUS_ARCHIVED = 'status: archived';
+const STATUS_WAITING = 'status: waiting on response';
+
+const FLAG_IGNORE = 'flag: bot ignore';
+
+const LABELS_FOR_NEXT_MILESTONE = [
+  TYPE_BUG,
+  TYPE_DOCS,
+];
+
+const LABELS_FOR_BACKLOG = [
+  TYPE_CI,
+  TYPE_CODE_HEALTH,
+  TYPE_ENHANCEMENT,
+  TYPE_PERFORMANCE,
+];
+
+const PING_QUESTION_TEXT =
+    'Does this answer all your questions? ' +
+    'If so, would you please close the issue?';
+
+const CLOSE_STALE_TEXT =
+    'Closing due to inactivity. If this is still an issue for you or if you ' +
+    'have further questions, the OP can ask shaka-bot to reopen it by ' +
+    'including `@shaka-bot reopen` in a comment.';
+
+const PING_INACTIVE_QUESTION_DAYS = 4;
+const CLOSE_AFTER_WAITING_DAYS = 7;
+const ARCHIVE_AFTER_CLOSED_DAYS = 60;
+
+
+async function archiveOldIssues(issue) {
```

Comment:
All of these methods are called somethingIssues, but they are operations on a single issue. They should probably be renamed.

Reply:
I disagree.  They accept a single issue at a time, so that the loop over issues is somewhat more efficient, but they are inherently filtering through issues (plural).  If you try to describe what a function does in terms of a single issue, you end with much wordier names like "archiveIssueIfOld".